monai/metrics/froc.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2020 - 2021 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import List, Optional, Tuple, Union
+
+import numpy as np
+import torch
+
+
+def compute_fp_tp_probs(
+    probs: Union[np.ndarray, torch.Tensor],
+    ycorr: Union[np.ndarray, torch.Tensor],
+    xcorr: Union[np.ndarray, torch.Tensor],
+    evaluation_mask: Union[np.ndarray, torch.Tensor],
+    isolated_tumor_cells: Optional[List] = None,
```

Comment:
this `isolated_tumor_cells` option is too task-specific, could be `target_labels`?